lib/util/check-unsupported-builtins.js

```diff
@@ -42,15 +43,15 @@ function isSupported({ supported }, configured) {
             ...supported.map(
                 version => `>= ${version} < ${major(version) + 1}`
```

Comment:
Also, thinking about this, could this just be:
```suggestion
                version => `^${version}`
```